### PR TITLE
Fix -t none in afl-showmap

### PR DESCRIPTION
**Pull request by edmcman**

### CODE REVIEW

File: src/afl-showmap.c [hunk 1]
```diff
@@ -1015,7 +1015,7 @@ int main(int argc, char **argv_orig, char **envp) {
 
           }
 
-        }
+        } else fsrv->exec_tmout = 0;
```
Comment (vanhauser-thc) [inline src/afl-showmap.c]: can you please put `{}` around it? thank you

**Comment by vanhauser-thc** [comment]
thank you, I wasnt even aware that there is a -t none variant :)

**Comment by edmcman** [comment]
Me neither.  But cmin uses that as the default unless you override it.

**Comment by edmcman** [comment]
My change doesn't work for the fork server.  I am looking into it.  I think it has to do with read_s32_timed.

**Comment by vanhauser-thc** [comment]
I think -t none should rather translate to -t 120000 (120 seconds). it doesnt make sense to allow infinity.

**Comment by edmcman** [comment]
Yeah, I was leaning in that direction, or just changing the default in afl-cmin to be 1000 (or 120000).

**Comment by vanhauser-thc** [comment]
are you still on this? :)

**Comment by edmcman** [comment]
This should be ready to go once the CI finishes.  It treats `-t none` as `-t 120000`

**Comment by vanhauser-thc** [comment]
thanks! I will update the help though.

**Pull request merged**